presto-spi/src/main/java/com/facebook/presto/spi/statistics/StatisticsValue.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.facebook.presto.spi.statistics;
+
+import static java.util.Objects.requireNonNull;
+
+public class StatisticsValue<T>
+{
+    private final Reliability reliability;
```

Comment:
As we discussed if it should be moved somewhere it would be column (not table) level.
As we may have different level of knowledge for different columns.
Also let's keep it here for now. And see if it is painful. Theoretically we could have different level of reliability for different stats (e.g. min/max vs ndv)